packages/backend/index.ts

```diff
@@ -0,0 +1,112 @@
+import express, { Express, Request, Response, Application, NextFunction } from 'express';
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
+import logger from 'morgan';
+import createError from 'http-errors';
+import path from 'path';
+import Knex from 'knex';
+import knexConfig from './knexfile'
+import { Model } from 'objection'
+import dotenv from 'dotenv';
+import passport from 'passport';
+import session from 'express-session';
+import { Strategy as GoogleStrategy } from 'passport-google-oauth20';
+
+import User from './user/user';
+import authRouter from './routes/auth';
+import indexRouter from './routes/index';
+import usersRouter from './routes/users';
+
+//For env File 
+dotenv.config({ path: __dirname + '/.env.local' });
+
+// Initialize knex.
+const knex = Knex(knexConfig.development)
+Model.knex(knex)
+
+const app: Application = express();
+const port = process.env.PORT || 3001;
+
+app.use(
+    cors({
```

Comment:
Nit but spacing is kinda inconsistent between `backend` and `frontend`, might wanna set up Prettier

Reply:
good catch. setup prettier in backend and blended prettier with eslint for the frontend